src/go/rpk/pkg/cli/cmd/redpanda/config_test.go

```diff
@@ -133,6 +137,13 @@ func TestBootstrap(t *testing.T) {
 			conf, err := new(config.Params).Load(fs)
 			require.NoError(t, err)
 
+			if tt.id != "" {
+				require.NotNil(t, conf.Redpanda.ID)
+				require.Equal(t, 1, *conf.Redpanda.ID)
```

Comment:
should it better be compared to the `tt.id` value?

Reply:
Done